preset/inference-optimization/docker-compose.yaml

```diff
@@ -54,6 +54,8 @@ services:
         https_proxy: ${https_proxy}
         no_proxy: ''
       context: ../deep-learning
+      labels:
```

Comment:
I removed the labels for dl-base because you're not publishing them

Reply:
got it, thanks!